tests/one_to_one/models.py

```diff
@@ -118,3 +118,18 @@ class Director(models.Model):
     is_temp = models.BooleanField(default=False)
     school = models.OneToOneField(School)
     objects = DirectorManager()
+
+
+class UnsavedOneToOneField(models.OneToOneField):
+    # A OneToOneField which can point to an unsaved object
+    unsaved_instances_allowed = True
```

Comment:
Thanks @timgraham. I've changed the name and added a note to releases/1.8. Shall I add further documentation to https://docs.djangoproject.com/en/1.7/ref/models/fields/ ?